ingestion/src/metadata/ingestion/ometa/provider_registry.py

```diff
@@ -65,3 +66,8 @@ def azure_auth_init(config: OpenMetadataConnection) -> AuthenticationProvider:
 @auth_provider_registry.add(AuthProvider.custom_oidc.value)
 def custom_oidc_auth_init(config: OpenMetadataConnection) -> AuthenticationProvider:
     return CustomOIDCAuthenticationProvider.create(config)
+
+
+@auth_provider_registry.add(AuthProvider.openmetadata.value)
+def om_auth_init(config: OpenMetadataConnection) -> AuthenticationProvider:
+    return OpenMetadataAuthenticationProvider.create(config)
```

Comment:
Hi @harshach , this LGTM. We'd just be missing the entry for the lineage backend here https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/airflow_provider_openmetadata/lineage/config/providers.py

Reply:
@pmbrull thanks for the catch. Fixed it.